docs/concepts.rst

```diff
@@ -16,9 +16,14 @@ The Nonprofit Success Pack and the Product Delivery Model
 
 The Nonprofit Success Pack, or NPSP, is one of Salesforce.org's flagship products. The NPSP consists of six managed packages, with complex dependency relationships between them, alongside unpackaged metadata and setup automation that helps create a usable NPSP org.
 
-Salesforce.org delivers NPSP using CumulusCI and the Product Delivery Model because those tools are what make it possible for us to ship this large, complex, heterogeneous application efficiently to tens of thousands of customers — without a lengthy setup guide to be completed by the end user. CumulusCI automation seamlessly deploys all six managed packages in the right sequence, delivers the unpackaged metadata, like Record Types and Page Layouts, we supply to make the customer's life easier, and executes setup automation like populating NPSP relationship values. 
+Salesforce.org delivers NPSP using CumulusCI and the Product Delivery Model because those tools are what make it possible for us to ship this large, complex, heterogeneous application efficiently to tens of thousands of customers — without a lengthy setup guide to be completed by the end user. CumulusCI automation seamlessly
 
-CumulusCI runs this automation throughout our development lifecycle, starting from feature branches in the hands of our developers and culminating the delivery of new versions of the application to our users, including through MetaDeploy — which runs the very same automation we use internally to set up and configure a customer org.
+* deploys NPSP's five dependent managed packages in the right sequence;
+* delivers unpackaged Record Types for the Account and Opportunity objects;
+* installs NPSP;
+* and performs final configuration to make the customer's experience better, like setting up Global Actions and delivering translations.
+
+CumulusCI runs this automation throughout our development lifecycle, starting from feature branches in the hands of our developers and culminating in the delivery of new versions of the application to our users through MetaDeploy.
 
 Throughout the CumulusCI documentation, we'll have the Product Delivery Model in mind. 
 
```

Comment:
Before we tackle Projects, I think it's an opportune time to do a thorough retread of MANAGED and UNMANAGED PACKAGES. These are what draw users to CumulusCI in the first place, so let's nail that point home.